charts/monasca/values.yaml

```diff
@@ -279,4 +279,16 @@ zookeeper:
     accessMode: ReadWriteOnce
     size: 10Gi
 
-
+client:
+  name: client
+  enabled: false
+  image:
+    repository: rbrndt/python-monascaclient
```

Comment:
Probably a good idea eventually, we can probably give this image the same treatment as the others and slim it down while we're at it... it's currently just under 300 MB compressed: https://hub.docker.com/r/rbrndt/python-monascaclient/tags/